.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.vscode/
```

Comment:
This is something you could add to `~/.gitignore` on your machine, to never commit these files. Then we don't need it specifically in this repo.